CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 0.5.1 (06/01/2016)
+
+### Changes
+
+* Change header name emitted by `Client#apply` from "Authorization" to "authorization" ([@murgatroid99][])
+
```

Comment:
FYI: this has been released.

On 6 January 2016 at 11:04, Michael Lumish <notifications@github.com> wrote:

> In CHANGELOG.md
> <https://github.com/google/google-auth-library-ruby/pull/58#discussion_r48994916>
> :
>
> > @@ -1,3 +1,9 @@
> > +## 0.5.1 (06/01/2016)
> > +
> > +### Changes
> > +
> > +* Change header name emitted by `Client#apply` from "Authorization" to "authorization" ([@murgatroid99][])
> > +
>
> done
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/google/google-auth-library-ruby/pull/58/files#r48994916>
> .
>



-- 
This email may be confidential or privileged.  If you received this
communication by mistake, please don't forward it to anyone else (it may
contain confidential or privileged information), please erase all copies of
it, including all attachments, and please let the sender know it went to
the wrong person. Thanks.